lib/datadog/appsec/contrib/rest_client/request_patch.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Datadog
+  module AppSec
+    module Contrib
+      module RestClient
+        # Module that adds SSRF detection to RestClient::Request#execute
+        module RequestPatch
```

Comment:
IMHO naming should reflect "what kind of patch" instead of saying "what i'm patching" because I can see what we are patching in the code, but I can't tell by name RequestPatch what it's doing

Reply:
Renamed [49978b9](https://github.com/DataDog/dd-trace-rb/pull/4424/commits/49978b97738e50394519973a1aa8adbe3359ec68)